docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '3'
+volumes:
+  dbvolume:
```

Comment:
`dbvolume` we should rename to `persistant_dir` and we should put RabbitMQ and Postgres storage in there